x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -54,6 +59,8 @@ export { Plugin };
 export * from './plugin';
 // TODO remove this import when we expose the Rules tables as a component
 export { loadRules } from './application/lib/rule_api/rules';
+export { loadRuleTypes } from './application/lib/rule_api';
```

Comment:
This is going to blow up our metrics, that's why we need to create shareable component to avoid it